src/resources/vue/components/ItemCard.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div class="item-card">
+        <div class="left">
+            <svg
```

Comment:
Moet deze svg niet variabel zijn? Dat je hier een image kan plaasten van het item?

Reply:
Kunnen we icons niet beter gewoon allemaal in 1 component zetten met een switch statement ofzo? Anders idd gwn in de slot douwen

Reply:
Denk dat een component beter is ja, met een switch statement